examples/Application/Traffic-Light-Control/ddqn.py

```diff
@@ -0,0 +1,95 @@
+#   Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
Can we use the parl.algorithms.DDQN directly?

Reply:
There are some tricks used in the `ddqn.py`, such as `grad clip`, `epsilon decay` ,`lr_decay`, which don't use in the `parl.algorithms.DDQN `. If using the `parl.algorithms.DDQN` directly, maybe all the experiments should be run again to make sure that `parl.algorithms.DDQN` performs well.